lib/ActivityStream.jsm

```diff
@@ -170,6 +170,10 @@ const PREFS_CONFIG = new Map([
   ["asrouter.snippetsUrl", {
     title: "A custom URL for the AS router snippets",
     value: ""
+  }],
```

Comment:
seems to need to be here and in the prerender, else it gets erased

Reply:
You can remove this with the changes mentioned above